test/e2e_node/criproxy/proxy_image_service.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package criproxy
+
+import (
+	"context"
+
+	kubeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
```

Comment:
The func names are synced from the ImageService https://github.com/kubernetes/kubernetes/blob/55b83c92b3b69cd53d5bf22b8ccff859a005241a/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto#L145-L161.

I'm not sure if this warrants a comment here.